tool/site/client/src/components/claim/createDispenseClaim.ts

```diff
@@ -52,7 +53,12 @@ export function createClaim(
   const finalMedicationDispense = medicationDispenses[medicationDispenses.length - 1]
   const prescriptionStatusExtension = getTaskBusinessStatusExtension(finalMedicationDispense.extension)
 
-  const contained = [tempPractitionerRole]
+  const containedPractitionerRole = medicationDispenses[0].contained
+    ?.find(resource => resource?.resourceType === "PractitionerRole") as fhir.PractitionerRole
+
+  const containedOrganization = dispensingOrganization
```

Comment:
we don't need this const here we can just pass `dispensingOrganization` directly

Reply:
done